packages/core/src/components/widgets/AltDateWidget.tsx

```diff
@@ -88,7 +100,7 @@ class AltDateWidget extends Component {
     );
   };
 
-  setNow = (event) => {
+  setNow = (event: MouseEvent) => {
```

Comment:
Can you try `MouseEvent<HTMLInputElement>` and see if that works?

Reply:
HTMLAnchorEvent works, added that.